ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml

```diff
@@ -245,6 +245,80 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-conformance,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
@alvaroaleman do we really want this to always run? Not sure if it was intentional...

Reply:
/hold

Reply:
Oh you are right, we don't want that

Reply:
@ironcladlou , @alvaroaleman ,thanks. I would assume it should always run. What do you suggest instead?

Reply:
/cc @davidvossel

Reply:
I would assume we would NOT always run this, and that users should opt-in. The tests consume resources, kubevirt tests aren't complete, aren't actually gating or informing for this team at the moment, etc.

Reply:
I doubt the value of always running these in hypershift repo PRs right now justifies the resource costs, but I guess that's up to whomever is paying for them...

Reply:
> I would assume we would NOT always run this, and that users should opt-in. The tests consume resources, kubevirt tests aren't complete, aren't actually gating or informing for this team at the moment, etc.

My thought would have been, that once https://github.com/openshift/hypershift/pull/822 is merged, we test our current progress and that we don't want regressions on that. Even if it is not incomplete. I would expect this PR to get merged after 822 is in, so that we can rehearse it here with the code in main.

What do you think you would need to feel comfortable with making it a gating requirement?

Reply:
It's easy to turn them on later if we ever wanted to for some reason. I vote we make `always_run: false`, merge this, and move on

Reply:
We don't even gate PRs on our full AWS suite or OCP conformance tests for the MVP platforms that we are trying to ship and support, so I can't really say from our side that this is a priority for us that justifies gating in terms of our interest in regressions. We've deemed periodics sufficient for those.

On the resource consumption side, I guess I don't care IF these tests do not consume resources shared by tests for the platforms we ARE trying to support right now, and as long as resources for these don't affect this team's resource accounting in any way. If it's taking (e.g.) azure clusters from a pool that our team doesn't use or rely on, it's not as much of a concern for me.

Reply:
changed `always_run` to `false`

Reply:
Looks like you changed it for the `pull-ci-openshift-hypershift-main-unit` job instead of `pull-ci-openshift-hypershift-main-e2e-kubevirt-all`?

Reply:
oops... fixed

Reply:
> We don't even gate PRs on our full AWS suite or OCP conformance tests for the MVP platforms that we are trying to ship and support, so I can't really say from our side that this is a priority for us that justifies gating in terms of our interest in regressions. We've deemed periodics sufficient for those.

Thanks for the details.